erpnext/accounts/doctype/sales_invoice/sales_invoice.js

```diff
@@ -783,3 +783,11 @@ var select_loyalty_program = function(frm, loyalty_programs) {
 
 	dialog.show();
 }
+frappe.ui.form.on('Sales Invoice', {
+	cost_center: function(frm, cdt, cdn) {
+		erpnext.utils.copy_parent_value_in_all_row(frm.doc, cdt, cdn, "items", "cost_center", "cost_center");
+		erpnext.utils.copy_parent_value_in_all_row(frm.doc, cdt, cdn, "taxes", "cost_center", "cost_center");
```

Comment:
same as purchase invoice comment?

Reply:
The purpose of copying parent cost center to item is to book sales against cost center selected in parent document to maintain cost center wise receivable.

Reply:
as per purchase invoice

Reply:
Okay, I will remove function to copy parent CC to child.